website/docs/guides/snowflake-qs.md

```diff
@@ -230,6 +230,23 @@ Now that you have a repository configured, you can initialize your project and s
         ```
     - In the command line bar at the bottom, enter `dbt run` and click **Enter**. You should see a `dbt run succeeded` message.
 
+:::warning
+If you receive an insufficient privileges error on Snowflake at this point, it may be because your Snowflake role doesn't have permission to access the raw source data, to build target tables and views, or both. To troubleshoot, use a role with sufficient privileges (like `ACCOUNTADMIN`) and run the following commands in Snowflake:
+
+```
+grant all on database raw to role <snowflake_role>;
+grant all on database analytics to role <snowflake_role>;
+
+grant all on schema raw.jaffle_shop to role <snowflake_role>;
+grant all on schema raw.stripe to role <snowflake_role>;
+
+grant all on all tables in database raw to role <snowflake_role>;
+grant all on future tables in database raw to role <snowflake_role>;
+```
+
+Note to replace `<snowflake_role>` with the role you intend to use. If you launched dbt Cloud with Partner Connect, then this role will be `pc_dbt_role`.
```

Comment:
@boxysean wanted to confirm pc_dbt_role is the actual role the user should use and isn't a placeholder

```suggestion
Note, replace `<snowflake_role>` with the role you intend to use. If you launched dbt Cloud with Snowflake Partner Connect, use `pc_dbt_role` as the role.
```